cmd/check.go

```diff
@@ -13,3 +18,25 @@ var checkCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(checkCmd)
 }
+
+func checkJunitAndWrite(cfg runtime.Config, results runtime.Results) error {
```

Comment:
I believe having separate methods for check and write will improve the code maintenance and readability.

Reply:
They do the exact same thing. There is no need to separate them at the moment.